README.md

```diff
@@ -12,12 +20,16 @@ Documentation of the actually implemented smart contract is generated automatica
 * [BaseDAO](https://github.com/tqtezos/baseDAO/blob/autodoc/master/BaseDAO.md)
```

Comment:
Probably it should be changed to `TrivialDAO` as well?

Reply:
Hm, true